packages/optimizers/htmlnano/src/HTMLNanoOptimizer.js

```diff
@@ -13,7 +13,10 @@ export default (new Optimizer({
   async loadConfig({config, options}) {
     let userConfig = await config.getConfigFrom(
       path.join(options.entryRoot, 'index.html'),
-      ['.htmlnanorc', '.htmlnanorc.js'],
+      ['htmlanorc.json', 'htmlnanorc.js'],
```

Comment:
Hmm we've supported .htmlnanorc for a long time and a bunch of projects might use it. Should we still support it in addition to the new ones? Should we deprecate it and have a warning?

Reply:
We can wait until parcel 3 to remove it if you want

Reply:
Not sure. Let's discuss with the team. Another option is to detect the file but make it an error so people know to rename it when upgrading.

Reply:
The README mentions these, but I don't think it's an exhaustive list. (I believe it's all the names that [cosmicconfig](https://github.com/davidtheclark/cosmiconfig) supports.)

Reply:
Ah you're right. According to [this](https://github.com/davidtheclark/cosmiconfig/blob/main/README.md#cosmiconfig) it should be:

* package.json#htmlnano
* `.htmlnanorc`
* `.htmlnanorc.json`
* `.htmlnanorc.yml`
* `.htmlnanorc.yaml`
* `.htmlnanorc.js`
* `.htmlnanorc.cjs`
* `htmlnano.config.js`
* `htmlnano.config.cjs`